util/getSkills.js

```diff
@@ -0,0 +1,34 @@
+// order of keys in this array determines the order of the return array 
+const skillKeys = [
+  'technologies',
+  "expertise",
+  "strengths",
+  "certifications",
+  "interests",
+];
+const getSkillIndex = (skill) => skillKeys.indexOf(skill);
+
+// conditional logic layer
+const isArray = (data) => Array.isArray(data);
+const isSkillKey = (key) => skillKeys.includes(key);
+
+// data transform layer 
+const removeDuplicateArrayValues = (values) =>[...new Set(values)];
```

Comment:
Like the use of Set() ~ @dustin-jw made a good recommendation. Glad you chose to use it here!